IDE_Board_Manager/prusa3dboards/cores/prusa_einsy_rambo/CDC.cpp

```diff
@@ -34,14 +34,14 @@ typedef struct
 static volatile LineInfo _usbLineInfo = { 57600, 0x00, 0x00, 0x00, 0x00 };
 static volatile int32_t breakValue = -1;
 
-bool _updatedLUFAbootloader = false;
+static u8 wdtcsr_save;
 
 #define WEAK __attribute__ ((weak))
 
 extern const CDCDescriptor _cdcInterface PROGMEM;
 const CDCDescriptor _cdcInterface =
 {
-	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,1),
+	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,0),
```

Comment:
What is this change 1->0 in the USB CDC descriptor?

Reply:
We are not using anything about usb anyway. Even the mmu board def uses the builtin arduino core, not this custom core

Reply:
If someone uses Arduino IDE 1.8.19 to compile the MMU 1.06 FW it will use same `CDC.cpp` as this is a copy of the Arduino IDE `hardware/arduino/avr/cores/arduino`

But normally the MMU2 1.0.6 should be compiled with https://github.com/prusa3d/MM-build-env by https://github.com/prusa3d/MM-control-01/blob/master/build.sh#L12

Reply:
ok, it doesn't look like being used in our FW, so hopefully no hidden harm done at this spot